lib/rubocop/cop/rspec/single_argument_message_chain.rb

```diff
@@ -22,22 +22,42 @@ class SingleArgumentMessageChain < Cop
           (send _ #{Matchers::MESSAGE_CHAIN.node_pattern_union} $...)
         PATTERN
 
+        def_node_matcher :hash?, '(hash ...)'
+
         def on_send(node)
           message_chain(node) do |(first, *remaining)|
             return if first.to_s.include?('.') || remaining.any?
 
+            return if hash?(first) && first.children.count != 1
```

Comment:
You could, alternatively, just match a hash with a single pair. I think it might just be `'(hash pair)'`